docs/user_guide/configuring.rst

```diff
@@ -378,14 +378,16 @@ Add a JSON file to define your switcher's versions
 
 First, write a JSON file stating which versions of your docs will be listed in
 the switcher's dropdown menu. That file should contain a list of entries that
-each have one or two fields:
+each can have the following fields:
 
 - ``version``: a version string. This will be inserted into
   ``switcher['url_template']`` to create the links to other docs versions, and
   also checked against ``switcher['version_match']`` to provide styling to the
   switcher.
 - ``name``: an optional name to display in the switcher dropdown instead of the
   version string (e.g., "latest", "stable", "dev", etc).
+- ``url``: an optional URL. If provided, it links the version to ``url``
```

Comment:
Could you add an example of this usage in one of the entries in the example below, with a comment like "Note that because `url` is specified for entry XXX, this URL will be used instead of the automatically-generated one."

Reply:
I've added it in an example. But since it's a JSON file I cannot really add a comment no??